### DISCUSSION TITLE
Is Composer required to install OpenMage v20?

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#3892] [Questions & Answers]
Is there another option available not using this composer. I can remember testing a bit with this adobe magento 2.3 and 2.4 and there the composer was using >1GB of memory. Looks a bit like shitty coding, requiring so much memory for ... just dependency checking.

**Comment by fballiano** [comment]
you can just download this file as it is built with all dependencies:
<img width="548" alt="Screenshot 2024-03-24 alle 20 26 27" src="https://github.com/OpenMage/magento-lts/assets/909743/b9c72073-2668-463f-8369-d081430bc330">

**Reply by f1-outsourcing** [reply]
:) thanks!

**Comment by f1-outsourcing** [comment]
Maybe better not to store permissions in zip file? I don't have an option to turn them of like with tar --no-same-permissions. I am not really using the others permission on my environment.
